test-vectors/did_dht/create.json

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
i was under the impression that we want to have 1 JSON per test vector, so we can do something like
```
read 1 test vector file -> parse 1 JSON object -> testCase(JSON)
```

with this structure, how do you parse out and use for multiple test cases?

Reply:
for the web5-* we have formatting like this:
https://github.com/TBD54566975/web5-spec/blob/main/test-vectors/README.md

you take the array of test vectors, they will all attempt to parse in the object and run the feature

Reply:
will want to consolidate and have tbdex be the same, otherwise we will have 100s of json files

Reply:
Actually after discussion with Diane it makes sense to leave tbdex vectors the way they are